### PR TITLE
LIVE-448 - Add playwright test suite for market page

### PR DESCRIPTION
**Pull request by thomasrogerlux**
## 🦒 Context (issues, jira)

[LIVE-448]

## 💻  Description / Demo (image or video)

Add test suite for market page (previous PR https://github.com/LedgerHQ/ledger-live-desktop/pull/4704 got closed by release)

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

[LIVE-448]: https://ledgerhq.atlassian.net/browse/LIVE-448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by thomasrogerlux** [comment]
Wait for https://github.com/LedgerHQ/ledger-live-common/pull/1728

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@ledgerhq/hw-transport-http": "6.24.1",
     "@ledgerhq/hw-transport-node-hid-singleton": "6.24.1",
     "@ledgerhq/ledger-core": "6.14.5",
-    "@ledgerhq/live-common": "^21.32.5",
+    "@ledgerhq/live-common": "https://github.com/LedgerHQ/ledger-live-common.git#develop",
```
Comment (LFBarreto) [inline package.json:52]: could you update this :pleading_face:

**Review by LFBarreto — CHANGES_REQUESTED** [review]
just update the llc version and were good :+1:

**Review by LFBarreto — APPROVED** [review]

**Comment by thomasrogerlux** [comment]
generate-screenshots

**Review by nabil-brn — APPROVED** [review]
Good to go, no new screenshots, the tests should not be flaky anymore once we merged this PR.

File: tests/specs/market.spec.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    expect(await page.screenshot()).toMatchSnapshot("market-page-no-scrollbar.png");
+  });
+
+  await test.step("inverse sorting", async () => {
+    await marketPage.toggleInvertSort();
+    await marketPage.waitForLoading();
+    expect(await page.screenshot()).toMatchSnapshot("market-page-inverted-sort.png");
+    await marketPage.toggleInvertSort();
+    await marketPage.waitForLoading();
+    expect(await page.screenshot()).toMatchSnapshot("market-page.png");
```
Comment (valpinkman) [inline tests/specs/market.spec.ts:27]: only one expect per test 🙏🏻

**Review by valpinkman — CHANGES_REQUESTED** [review]
Except for the splitting of the one test I mentionned, everything looks fine to me. Fix this and we will be ready ;)

File: tests/specs/market.spec.ts (hunk 2 above)
Comment (thomasrogerlux) [reply tests/specs/market.spec.ts:30]: Done!

**Review by valpinkman — APPROVED** [review]

**Review by nabil-brn — APPROVED** [review]

**Pull request merged**